1.1: ARMSVE fails to build on Windows

The runtime detection of the presence of the CPU feature uses a method not available in Windows.
```
Compiling obj/arm64/kernels/armsve/1m/bli_dpackm_armsve256_int_8xk.o ('armsve' CFLAGS for kernels)
../blis-1.1/config/armsve/bli_cntx_init_armsve.c:36:10: fatal error: 'sys/auxv.h' file not found
   36 | #include <sys/auxv.h>
      |          ^~~~~~~~~~~~
1 error generated.
make: *** [Makefile:707: obj/arm64/config/armsve/bli_cntx_init_armsve.o] Error 1
make: *** Waiting for unfinished jobs....
==> ERROR: A failure occurred in build().
```
A Windows compatible way of finding the same needs to be added.

BLIS 1.1

@devinamatthews correct. Or at least `firestorm`. 

We ended up ignoring the `armsve` sub-configuration when using `arm64` as the configuration in a patch at 

I tried your PR, it's getting past the original issue, but now linking doesn't work and I think it's a bug in Clang:
```
Compiling obj/arm64/ref_kernels/armsve/1/bli_xpbyv_armsve_ref.o ('armsve' CFLAGS for ref. kernels)
Compiling obj/arm64/ref_kernels/armsve/1f/bli_axpy2v_armsve_ref.o ('armsve' CFLAGS for ref. kernels)
Compiling obj/arm64/ref_kernels/armsve/1f/bli_axpyf_armsve_ref.o ('armsve' CFLAGS for ref. kernels)
Compiling obj/arm64/ref_kernels/armsve/1f/bli_dotaxpyv_armsve_ref.o ('armsve' CFLAGS for ref. kernels)
Compiling obj/arm64/ref_kernels/armsve/1f/bli_dotxaxpyf_armsve_ref.o ('armsve' CFLAGS for ref. kernels)
Compiling obj/arm64/ref_kernels/armsve/1f/bli_dotxf_armsve_ref.o ('armsve' CFLAGS for ref. kernels)
Compiling obj/arm64/ref_kernels/armsve/1m/bli_packm_cxc_diag_1er_armsve_ref.o ('armsve' CFLAGS for ref. kernels)
fatal error: Incorrect size for bli_caxpyf_armsve_ref epilogue: 48 bytes of instructions in range, but .seh directives corresponding
      to 44 bytes

1 error generated.
Compiling obj/arm64/ref_kernels/armsve/1m/bli_packm_cxc_diag_armsve_ref.o ('armsve' CFLAGS for ref. kernels)
make: *** [Makefile:720: obj/arm64/ref_kernels/armsve/1f/bli_axpyf_armsve_ref.o] Error 1
make: *** Waiting for unfinished jobs....
Compiling obj/arm64/ref_kernels/armsve/1m/bli_packm_cxk_1er_armsve_ref.o ('armsve' CFLAGS for ref. kernels)
Compiling obj/arm64/ref_kernels/armsve/1m/bli_packm_cxk_armsve_ref.o ('armsve' CFLAGS for ref. kernels)
Compiling obj/arm64/ref_kernels/armsve/1m/bli_unpackm_cxk_armsve_ref.o ('armsve' CFLAGS for ref. kernels)
fatal error: Incorrect size for bli_zdotxaxpyf_armsve_ref epilogue: 48 bytes of instructions in range, but .seh directives
      corresponding to 44 bytes

1 error generated.
make: *** [Makefile:720: obj/arm64/ref_kernels/armsve/1f/bli_dotxaxpyf_armsve_ref.o] Error 1
==> ERROR: A failure occurred in build().
    Aborting...
```

--- a/config/armsve/bli_cntx_init_armsve.c
+++ b/config/armsve/bli_cntx_init_armsve.c
@@ -33,21 +33,24 @@
 */
 
 #include "blis.h"
-#include <sys/auxv.h>
-
-#ifndef HWCAP_SVE
-#define HWCAP_SVE (1 << 22)
-#endif
 
 void bli_cntx_init_armsve( cntx_t* cntx )
 {
-	if (!(getauxval( AT_HWCAP ) & HWCAP_SVE))
-		return;
-
-	blksz_t blkszs[ BLIS_NUM_BLKSZS ];
-
 	// Set default kernel blocksizes and functions.
 	bli_cntx_init_armsve_ref( cntx );
+
+	// If we are autodetecting the correct aarch64 config, then we have to make sure
+	// that SVE instructions are actually available since these are used in determining
+	// the register blocksizes.
+	#ifdef BLIS_FAMILY_ARM64
+	uint32_t family, model, features = 0;
+	bli_cpuid_query( &family, &model, &features );
+
+	if ( ! bli_cpuid_has_features( features, FEATURE_SVE ) )
+		return;
+	#endif
+
+	blksz_t blkszs[ BLIS_NUM_BLKSZS ];
 
 	// -------------------------------------------------------------------------
 

--- a/configure
+++ b/configure
@@ -2142,7 +2142,7 @@ check_assembler()
 
 check_os()
 {
-	if [[ "$(uname -s)" == "Darwin" && "$(uname -m)" == "arm64" ]]; then
+	if [[ ( "$(uname -s)" == "Darwin" || "$is_win" == "yes" ) && "$(uname -m)" == "arm64" ]]; then
 		blacklistos_add "armsve"
 	fi
 }

